packages/Python/lldbsuite/test/lang/swift/variables/bridged_string/main.swift

```diff
@@ -13,12 +13,12 @@ import Foundation
 
 func main()
 {
-	var s1 = "Hello world"
-	var s2 = "ΞΕΛΛΘ"
-	var s3 = s1 as NSString
-	var s4 = s2 as NSString
-	var s5 = "abc" as NSString
-	var s6 = String(s5)
+	let s1 = "Hello world"
+	let s2 = "ΞΕΛΛΘ"
```

Comment:
var's and let's are handled differently by the swift compiler.  I presume you changed this to remove the compiler warnings, but in tests like this we should really have a mix of var's and lets and we don't really care about the "you didn't use the var" warnings.  If the warnings bother you, just print all the variables at the end of the test...

Also, now that you know the String format, are each of the variants you had to cover below represented here?

Reply:
Yes, I changed this to appease the compiler, but I'll try something like '-w' if it bothers me in the future.

All the variants we need to cover are taken care of, but we will need separate handling for the 32-bit ABI.